azure-pipelines.yml

```diff
@@ -1,34 +1,40 @@
 # Azure DevOps pipeline to build, check source codes, run tests, and deploy.
 #
-# The following variables needs to be defined before running the deployment jobs 
-# defined in this pipeline:
+# To enable the check of source code with Danger JS you need to configure a valid
+# GitHub token by setting the following variable: 
+# - DANGER_GITHUB_API_TOKEN
+#
+# To enable the deployment in any environment you need to configure the following 
+# variable otherwise all the deployment jobs will be always skipped:
+# - DO_DEPLOY = true
+# in case of a manual run, you also need to configure the following additional 
```

Comment:
how's a manual run started ?

Reply:
@gunzip It is possible to use the Azure DevOps portal (the procedure will be documented), pressing the "Run pipeline" button after selecting the pipeline to run:

<img width="1176" alt="Screenshot 2020-03-02 at 14 05 34" src="https://user-images.githubusercontent.com/60641726/75679336-b8e02f00-5c8f-11ea-80b9-199596df4d53.png">

<img width="1430" alt="Screenshot 2020-03-02 at 14 07 07" src="https://user-images.githubusercontent.com/60641726/75679350-bda4e300-5c8f-11ea-937c-ad21706535cc.png">

you have the option to select the DevOps stage(s) to run (e.g. only "Deploy_production") and override the preconfigured values of one or more variables when (manually) running the pipeline (e.g. _PRODUCTION_ENABLE_MANUAL_DEPLOY = true_). 

**Note.** I defined additional variables and conditions in the _azure-pipelines.yml_ file to be sure that a manual run of the pipeline will execute deployment tasks in staging/production environment only when explicitly setting those variables (to avoid unwanted executions by mistake).

Reply:
can you please put all these informations into some documents in the [io-handbook](https://github.com/pagopa/io-handbook) ?